### PR TITLE
Feature/Alert inline - DAO-602 

### PR DESCRIPTION
**Pull request by Fabricevladimir**
- Added `alertInline` component
- Added info colors to both tailwind configs

### CODE REVIEW

**Review by RakeshUP — APPROVED** [review]
Looks good. Thanks

File: packages/ui-components/src/components/icons/interface/icon_info.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {IconType} from '..';
+
+export const IconInfo: IconType = ({height = 16, width = 16, ...props}) => {
+  return (
+    <svg
+      width={width}
+      height={height}
+      fill="none"
+      viewBox="0 0 16 17"
```
Comment (WissenIstNacht) [inline packages/ui-components/src/components/icons/interface/icon_info.tsx:10]: The 17 is kind of sus :D

**Review by WissenIstNacht — APPROVED** [review]
Solid work man, thanks.

**Review by sepehr2github — APPROVED** [review]
awesome as always thanks!

File: packages/ui-components/src/components/icons/interface/icon_info.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply packages/ui-components/src/components/icons/interface/icon_info.tsx:10]: It really is! I tried changing it to the usual 16, but it cut off the bottom of the icon. I have it in the issues to fix. I'll add it to the notion space for those.

**Pull request merged**